scripts/train/benchmark.sh

```diff
@@ -0,0 +1,169 @@
+# ------------------------------------------------------------
```

Comment:
Similarly might be good to add some explanation of what this is (overview of training commands).